cli/components/question.go

```diff
@@ -0,0 +1,31 @@
+package components
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/rs/zerolog/log"
+)
+
+func AskAYesNoQuestion(msg string) bool {
+	reader := bufio.NewReader(os.Stdin)
+
+	for {
+		fmt.Printf("%s [(y)es/(n)o]: ", msg)
```

Comment:
The output can be improved. Debian apt uses a simple approach:

```
Do you want to continue? [Y/n]
```

In addition the current implementation does not handle cases where users presses a non allowed value. Similar to debian, it should make clear what the default value is. I would recommend we just use the default pattern established by debian

Reply:
We improve that in a follow-up PR.